lib/policy_machine.rb

```diff
@@ -292,6 +292,14 @@ def accessible_objects(user_or_attribute, operation, options = {})
     end
   end
 
+  def accessible_objects_for_operations(user_or_attribute, operation, options = {})
```

Comment:
`operations` is plural in the method name but the param `operation` is singular.  Which should it be?

Reply:
oops, this was just a typo, should be plural.  prob because the singular is used often and i hit tab for autocomplete too hastily in my editor.  thanks for catching it!